src/Domain/ValueObject/EmailAddress.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the eluceo/iCal package.
+ *
+ * (c) 2021 Markus Poerschke <markus@poerschke.nrw>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Eluceo\iCal\Domain\ValueObject;
+
+use InvalidArgumentException;
+
+final class EmailAddress
+{
+    private string $emailAddress;
+
+    public function __construct(string $emailAddress)
+    {
+        if (!filter_var($emailAddress, FILTER_VALIDATE_EMAIL)) {
+            throw new InvalidArgumentException("$emailAddress is no valid e-mail address");
+        }
+
+        $this->emailAddress = $emailAddress;
+    }
+
+    public function getEmailAddress(): string
+    {
+        return $this->emailAddress;
+    }
+
+    public function toUri(): Uri
+    {
+        return new Uri('mailto:' . urlencode($this->emailAddress));
```

Comment:
Outlook 2016 is having trouble with the encoded "@" character in the email address.

According to RFC standard, the `mailto:` doesn't require the url-encoding of that character. From the [RFC 5545 3.8.4.3 (Organizer)](https://www.rfc-editor.org/rfc/rfc5545#section-3.8.4.3) section, see its examples, but it says "Value Type: CAL-ADDRESS" -> look for that in the RFC, find [definition here](https://www.rfc-editor.org/rfc/rfc5545#section-3.3.3) -> says "the value must be a mailto URI, as defined by [[RFC2368](https://www.rfc-editor.org/rfc/rfc2368)].", see its examples.

By RFC 2368's examples, it seems that it implies the "@" should remain unencoded. The solution then would mean parsing the email address, such as is seen in this related StackOverflow discussion on the topic: https://stackoverflow.com/questions/8940445/what-is-the-correct-way-to-escape-a-string-for-a-mailto-link

In our case, we're going to just look for `%40` in the `ORGANIZER` property and convert it back to `@`, which will suffice.